paasta_tools/marathon_serviceinit.py

```diff
@@ -138,20 +138,30 @@ def get_verbose_status_of_marathon_app(app):
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
     output.append("    Tasks:")
 
-    rows = [("Mesos Task ID", "Host deployed to", "Deployed at what localtime")]
+    rows = [("Mesos Task ID", "Host deployed to", "Deployed at what localtime", "Health checks")]
```

Comment:
Can you do "Health"
and then later do "Healthy / Unhealthy" to be consistent with the marathon dashboard?